### ISSUE TITLE
БЭМ ли это?

### ISSUE THREAD

**Issue by spone379** [bem-site/bem-forum-content-ru#1422]
С точки зрения БЭМ, правильно ли такое именование классов? Или новый блок не должен называться, как элемент предыдущего?

<section class="price-list">
        <h2 class="price-list__title">TITLE</h2>
        <div class="price-list__price-cards">
            <div class="price-cards__price-card">
                <h3 class="price-card__title">BEST VALUE</h3>
                <div class="price-card__content">
                    <h4 class="price-card__name">STUFF</h4>
                    <span class="price-card__price">25</span>
                </div>
            </div>
        </div>
    </section>

**Comment by tadatuta** [comment]
Все почти хорошо, не хватает только миксов с родительскими блоками:
```html
<section class="price-list">
    <h2 class="price-list__title">TITLE</h2>
    <div class="price-list__price-cards price-cards">
        <div class="price-cards__price-card price-card">
            <h3 class="price-card__title">BEST VALUE</h3>
            <div class="price-card__content">
                <h4 class="price-card__name">STUFF</h4>
                <span class="price-card__price">25</span>
            </div>
        </div>
    </div>
</section>
```

**Comment by spone379** [comment]
То есть, если называть блок, как элемент предыдущего блока то всегда нужны миксы? Если бы были другие названия, можно было и без них обойтись?

**Comment by tadatuta** [comment]
Да. Ожидается, что элементы не болтаются в воздухе, а относятся к какому-либо родительскому блоку.
